packages/cli/src/dbt/context.ts

```diff
@@ -15,16 +16,33 @@ export type DbtContext = {
 
 export const getDbtContext = async ({
     projectDir,
+    initialProjectDir,
 }: GetDbtContextArgs): Promise<DbtContext> => {
     const projectFilename = path.join(projectDir, 'dbt_project.yml');
     let config;
+
     try {
         config = yaml.load(
             await fs.readFile(projectFilename, { encoding: 'utf-8' }),
         ) as any;
-    } catch (e) {
+    } catch (e: any) {
+        if (projectDir !== '/') {
+            const parentDir = path.join(projectDir, '..');
+            console.log(
```

Comment:
I don't think the user needs to know this. If it's never found you could add to the error message that `dbt_project.yml` was not found in this directory or any of it's parent directories